src/Infrastructure/Services/OpenAiEmbeddingService.cs

```diff
@@ -88,16 +86,14 @@ public async Task<Vector> GetEmbedding(string inputString, string? apiKey)
 
                 return vector;
             }
-            else
-            {
-                if (result.Error == null)
-                {
-                    throw new Exception("Unknown Error");
-                }
 
-                Console.WriteLine($"{result.Error.Code}: {result.Error.Message}");
-                return null;
+            if (result.Error == null)
```

Comment:
You've removed an else here, this will be evaluated even when result.IsSuccessful

Reply:
We are returning the value `vector` if we enter the result.IsSuccessful block so the else is redundant